### DISCUSSION TITLE
VS Code server is now open source. How does this affects code-server ?

### DISCUSSION THREAD

**Discussion by abhay-ranawat** [coder/code-server#4455] [General] [answered]
Github : [Add server folder ](https://github.com/microsoft/vscode/commit/822f995357f725216cdee6e05f1f1552ccbbd882)

**Comment by abhay-ranawat** [comment]
@jsjoeio

**Comment by jsjoeio** [comment, marked as answer]
Thanks for the great question! We're aware of this, we're not sure what it means yet since this is not documented as an official feature, and there are some differences between Code upstream/code-server, notably the auth stuff. We are still actively maintaining code-server at this time and there are no changes to the project.

**Reply by abhay-ranawat** [reply]
Thanks for code-server! and for also for the reply.
